README.md

```diff
@@ -135,7 +136,7 @@ NoSSLv3   | When `true`, the SSLv3 protocol is disabled. If NoSSLv2 and NoSSLv3
 NoTLSv1_0 | When `true`, the TLSv1.0 protocol is disabled
 NoTLSv1_1 | When `true`, and if available on the platform, the TLSv1.1 protocol is disabled
 NoTLSv1_2 | When `true`, and if available on the platform, the TLSv1.2 protocol is disabled
-sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more information, see [OpenSSL's documentation](https://openssl.org/docs/manmaster/apps/ciphers.html "OpenSSL's documentation")
+sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more information, check `man ciphers` or see [OpenSSL's documentation](https://www.openssl.org/docs/manpages.html "OpenSSL's documentation")
```

Comment:
Why the non-standard GitHub-style markdown style ("OpenSSL's documentation")?

Also, this doesn't really address my concern. Clearly, OpenSSL moves their docs around. See my original comments.

Reply:
ho, that's previous style, will correct it.
will remove link and describe like "search internet for `openssl man cipher`".